### PR TITLE
Add Formatting for Code Snippets in the OpenAPI Guide

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add formatting for code snippets in the OpenAPI guide[1] in all versions.

[1] https://ballerina.io/learn/using-the-openapi-tools/

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: 1.1/learn/how-to-use-openapi-tools.md [hunk 1]
```diff
@@ -27,33 +27,43 @@ Code generation from OpenAPI to Ballerina can produce `ballerina mock services`
 For build time client stub generation, annotation support is provided.
 
 ### Mock service from OpenAPI
-`ballerina openapi gen-service <moduleName>:<serivceName> 
+
+```ballerina
```
Comment (pubudu91) [inline 1.1/learn/how-to-use-openapi-tools.md:31]:
```suggestion
```
```

Comment (pubudu91) [inline 1.1/learn/how-to-use-openapi-tools.md:31]: This should be just ```. Adding `ballerina` tells to format it as Ballerina code

Comment (pubudu91) [reply 1.1/learn/how-to-use-openapi-tools.md:31]: Needs to be fixed in other, non-ballerina code places too

Comment (praneesha) [reply 1.1/learn/how-to-use-openapi-tools.md]: Corrected.

**Pull request merged**